_source/lib/apollos.coffee

```diff
@@ -37,10 +37,11 @@ Apollos.user.login = {}
 ###
 Apollos.person = ->
 
-  userDoc = Apollos.user()
+  user = Apollos.user()
 
-  person = Apollos.people.findOne
-    userId: userDoc._id
+  if user.rock?.guid
```

Comment:
@johnthepink yes, we move, and @benwiley we will need to update the apollos-rock package to support the move

:goat: